packages/devextreme/js/ui/html_editor/modules/mentions.js

```diff
@@ -383,11 +388,13 @@ if(Quill) {
             return extend(super._getPopupConfig(), {
                 hideOnParentScroll: false,
                 onShown: () => {
+                    this._setAriaAttribute();
                     this._isMentionActive = true;
                     this._hasSearch = false;
                     this._focusFirstElement();
                 },
                 onHidden: () => {
+                    this.quill.root.removeAttribute('aria-activedescendant');
```

Comment:
Let's move this to created method and rename the method as `_toggleActiveDescendant`; Than u can call `
```javascript
this._toggleActiveDescendant(true);
// or
this._toggleActiveDescendant(false);
```